gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -118,6 +121,12 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             help="Root directory. Results, plots and metrics will be stored in subdirectories,"
             " e.g. {output_root}/results",
         )
+        parser.add_argument(
+            "--run_visualizer",
```

Comment:
nit: a shorter --run_viz might be better.

Reply:
fixed